frontend/src/components/QueueComponent/QueueTable/QueueTableDropdown/QueueTableDropdown.js

```diff
@@ -21,7 +21,7 @@ const QueueTableDropdown = props => {
     }
   }
   return (
-    <Select value={value} onChange={handleChange}>
+    <Select value={value} onChange={handleChange} fullWidth={true}>
```

Comment:
With boolean values that do not need to be  programmatically updated, just include them, instead of assigning them a value.

like this: https://github.com/CodeForPhilly/prevention-point/blob/3233440cb4ed9aa8ca124c22ae24bbfb5133e44c/frontend/src/components/ParticipantForm.js#L50

fix that and i will approve